resolver.go

```diff
@@ -438,3 +438,28 @@ func (r *queryResolver) Account(_ context.Context, address model.Address, projec
 
 	return adapter.AccountToAPI(acc), nil
 }
+
+func (r *queryResolver) ProjectList(ctx context.Context) (*model.ProjectList, error) {
+	user, err := r.auth.GetOrCreateUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	var projects []*model.Project
```

Comment:
This should be implemented in the controller. Resolvers are only meant to do authorization and calling adapters if needed.